faq/faq.rst

```diff
@@ -43,21 +43,21 @@ ingest workflows.
 Can I join distributed and non-distributed tables together in the same query?
 -----------------------------------------------------------------------------
 
-If you want to do joins between small dimension tables (regular Postgres tables) and large tables (distributed), then you could distribute the small tables by creating 1 shard and N (with N being the number of workers) replicas. Citus will then be able to push the join down to the worker nodes. If the local tables you are referring to are large, we generally recommend to distribute the larger tables to reap the benefits of sharding and parallelization which Citus offers. For a deeper discussion into the way Citus handles joins, please see our :ref:`joins` documentation.
+If you want to do joins between small dimension tables (regular Postgres tables) and large tables (distributed), then you can distribute the small tables as "reference tables." This creates a single shard replicated across all worker nodes. Citus will then be able to push the join down to the worker nodes. If the local tables you are referring to are large, we generally recommend to distribute the larger tables to reap the benefits of sharding and parallelization which Citus offers. For a deeper discussion, see :ref:`reference_tables` and our :ref:`joins` documentation.
 
-Are there any PostgreSQL features not supported by CitusDB?
------------------------------------------------------------
+Are there any PostgreSQL features not supported by Citus?
+---------------------------------------------------------
 
 Since Citus provides distributed functionality by extending PostgreSQL, it uses the standard PostgreSQL SQL constructs. This includes the support for wide range of data types (including semi-structured data types like jsonb, hstore), full text search, operators and functions, foreign data wrappers, etc.
 
-PostgreSQL has a wide SQL coverage; and Citus does not support that entire spectrum out of the box when querying distributed tables. Some constructs which aren't supported natively for distributed tables are:
+PostgreSQL has a wide SQL coverage; and Citus does not support that entire spectrum out of the box when querying distributed tables that span multiple worker nodes. Some constructs which aren't supported natively in this case are:
```

Comment:
Could we invert this? We start by saying Citus doesn't support these out of the box, and then later note we do for multi-tenancy. Could we start by saying we support all these for multi-tenant apps? And maybe a small paragraph around limited support for real-time?